apps/web/pages/api/auth/[...nextauth].tsx

```diff
@@ -449,7 +464,11 @@ export default NextAuth({
                 console.error("Error while linking account of already existing user");
               }
             }
-            return true;
+            if (existingUser.twoFactorEnabled) {
+              return `/auth/login?totp=${await signJwt({ email: existingUser.email })}`;
```

Comment:
This is te redirection needed to let the user input a totp code in an external provider use case.